subprojects/gradle/module-types/src/main/kotlin/com/avito/android/module_type/internal/FileSerializer.kt

```diff
@@ -0,0 +1,23 @@
+package com.avito.android.module_type.internal
+
+import java.io.File
+import java.io.FileInputStream
+import java.io.FileOutputStream
+import java.io.ObjectInputStream
+import java.io.ObjectOutputStream
+import java.io.Serializable
+
+internal object FileSerializer {
```

Comment:
I don't get where and how that is used

Reply:
It's used to write and read `Serializable` objects. In this case - `ModuleDescription`.

Tasks are wired through these files as output/input. See the picture in PR's description.